scs2-session-visualizer-jfx/src/main/java/us/ihmc/scs2/sessionVisualizer/jfx/controllers/yoPieChart/YoPieChartVariableController.java

```diff
@@ -0,0 +1,34 @@
+package us.ihmc.scs2.sessionVisualizer.jfx.controllers.yoPieChart;
+
+import javafx.fxml.FXML;
+import javafx.scene.chart.PieChart;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import us.ihmc.scs2.sessionVisualizer.jfx.managers.SessionVisualizerToolkit;
+import us.ihmc.yoVariables.variable.YoDouble;
+import us.ihmc.yoVariables.variable.YoVariable;
+
+public class YoPieChartVariableController extends YoPieChartInputController
```

Comment:
Here is the variable class that will extend the YoPieChartInputController. Not sure why we need this but it pattern matches from the other implementation in SCS@